api/envoy/service/auth/v3/external_auth.proto

```diff
@@ -69,6 +70,9 @@ message OkHttpResponse {
   // by Leaving `append` as false, the filter will either add a new header, or override an existing
   // one if there is a match.
   repeated config.core.v3.HeaderValueOption headers = 2;
+
+  // Optional response metadata that will be emitted as dynamic metadata.
```

Comment:
Which part of the dynamic metadata keyspace does this live?

Reply:
If I catch your question correctly: for HTTP filter, `HttpFilterNames::get().ExtAuthorization` (`envoy.filters.http.ext_authz`) while for the network one: `NetworkFilterNames::get().ExtAuthorization` (`envoy.filters.network.ext_authz`)

Reply:
I think we should document these namespaces in the proto where the `emit_dynamic_metadata` can be enabled. And perhaps refer to those configs here?

Reply:
Yeah, can we spell this out explicitly in the docs? Thanks. Otherwise LGTM for API.

Reply:
Sure!